mgmt/rest/v2/api.go

```diff
@@ -56,25 +56,277 @@ func New(wg *sync.WaitGroup, killChan chan struct{}, protocol string) *apiV2 {
+		//
+		// Consumes:
+		// application/json
+		// application/x-protobuf
+		//
+		// Produces:
+		// application/json
+		// application/x-protobuf
+		//
+		// Schemes: http, https
+		//
+		// Responses:
+		// 200: TasksResponse
 		api.Route{Method: "GET", Path: prefix + "/tasks", Handle: s.getTasks},
+
+		// swagger:route GET /tasks/{id} getTask
+		//
+		// lists a task by the giving task id. Otherwise a not found error returns.
+		//
+		// Consumes:
+		// application/json
+		// application/x-protobuf
+		//
+		// Produces:
+		// application/json
+		// application/x-protobuf
+		//
+		// Schemes: http, https
+		//
+		// Responses:
+		// 200: TaskResponse
+		// 404: ErrorResponse
 		api.Route{Method: "GET", Path: prefix + "/tasks/:id", Handle: s.getTask},
+		// swagger:route GET /tasks/{id}/watch watchTask
+		//
+		// watches a task data stream for the giving task id. Otherwise, an error returns.
+		//
+		// Consumes:
+		// application/json
+		// application/x-protobuf
+		// text/event-stream
+		//
+		// Produces:
+		// application/json
+		// application/x-protobuf
+		// text/event-stream
+		//
+		// Schemes: http, https
+		//
+		// Responses:
+		// 200: TaskResponse
+		// 404: ErrorResponse
+		// 500: ErrorResponse
 		api.Route{Method: "GET", Path: prefix + "/tasks/:id/watch", Handle: s.watchTask},
+		// swagger:route POST /tasks addTask
+		//
+		// creates a task based on the input. Othereise, an error returns if the input misses the required fields or is in a malformed format.
+		//
+		// Consumes:
+		// application/json
+		// application/x-protobuf
+		//
+		// Produces:
+		// application/json
+		// application/x-protobuf
+		//
+		// Schemes: http, https
+		//
+		// Responses:
+		// 201: TaskResponse
+		// 500: ErrorResponse
 		api.Route{Method: "POST", Path: prefix + "/tasks", Handle: s.addTask},
+		// swagger:route PUT /tasks/{id} updateTaskState
+		//
+		// updates a task's state for the giving task id and the input state. An error occurs for any bad request.
+		//
+		// Consumes:
+		// application/json
+		// application/x-protobuf
+		//
+		// Produces:
+		// application/json
+		// application/x-protobuf
+		//
+		// Schemes: http, https
+		//
+		// Responses:
+		// 204: TaskResponse
+		// 400: ErrorResponse
+		// 409: ErrorResponse
+		// 500: ErrorResponse
 		api.Route{Method: "PUT", Path: prefix + "/tasks/:id", Handle: s.updateTaskState},
+		// swagger:route DELETE /tasks/{id} removeTask
+		//
+		// deletes a task for the giving task id. Note that only a stopped task may be removed. Otherwise, an error occurs.
+		//
+		// Consumes:
+		// application/json
+		// application/x-protobuf
+		//
+		// Produces:
+		// application/json
+		// application/x-protobuf
+		//
+		// Schemes: http, https
+		//
+		// Responses:
+		// 204: TaskResponse
+		// 404: ErrorResponse
+		// 500: TaskErrorResponse
 		api.Route{Method: "DELETE", Path: prefix + "/tasks/:id", Handle: s.removeTask},
 	}
```

Comment:
This PR applies swagger to the V2 API which is the version we would like "new" integrations to use.